index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of second card -->
+        <div class="card-works">
+          <img
+            src="./images/work/Tonic2.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Tonic</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of third card -->
+        <div class="card-works">
+          <img
+            src="./images/work/multipost2.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Multi-post Stories</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of fourth card -->
+      </section>
+      <!-- End of Work Section -->
+
+      <!-- ABOUT MYSELF SECTION -->
+      <section id="aboutmyself">
```

Comment:
The design for this section is missing detail from the template design.

Here's your design:

<img width="246" alt="Screen Shot 2022-05-27 at 17 37 14" src="https://user-images.githubusercontent.com/82426682/170797049-caf1d2e4-dad2-4c5d-9473-b46c93530685.png">

Here's the template design:

<img width="281" alt="Screen Shot 2022-05-27 at 17 37 54" src="https://user-images.githubusercontent.com/82426682/170797099-8fbd20f4-4ef7-409f-a4bd-b46f562b3a5a.png">

Do you see that little `border-radius` in the top right corner of the template design? Please implement it in your design.